htlcswitch/interfaces.go

```diff
@@ -67,8 +67,8 @@ type ChannelLink interface {
 	// initially.
 	//
 	// NOTE: This function MUST be non-blocking (or block as little as
-	// possible).
-	HandleSwitchPacket(*htlcPacket) error
+	// possible) if the sync bool parameter is false.
```

Comment:
Should be a new sentence. Also no body at all in this commit message.

Reply:
Not applicable anymore

Reply:
Bodies added to this commit and others